packages/compass-app-stores/src/index.js

```diff
@@ -1,19 +0,0 @@
-import { InstanceStore } from './stores';
-/**
- * Activate all the components in the App Stores package.
- * @param {Object} appRegistry - The Hadron appRegisrty to activate this plugin with.
- **/
-function activate(appRegistry) {
-  appRegistry.registerStore('App.InstanceStore', InstanceStore);
```

Comment:
Looking at the test failures, there are a few places that access `App.InstanceStore'` from the registry. I guess we can temporarily switch them to use the global one we set on `hadron-app` until we refactor them to use service injection too